docs/src/fs.rst

```diff
@@ -430,6 +430,11 @@ API
 
     Equivalent to :man:`utime(2)`, :man:`futimes(3)` and :man:`lutimes(3)` respectively.
 
+    Passing `INFINITY` as the atime or mtime sets the timestamp to the
```

Comment:
How about having some UV_UTIME_NOW define so it reads a bit nicer?

Reply:
I was kind of hoping to avoid a `#include <math.h>` in uv.h but I agree it reads better so I'll make that change.